docs-site/src/content/docs/fr/reference/cli/agents.md

```
@@ -0,0 +1,249 @@
+
+Voir [Combos](/fr/guides/combos/) pour le comportement de routage et les conseils de configuration.
+
+## Observabilité et débogage
+
+### `ocx observe <logs|usage|storage|memory|debug|claude-inbound|injection> ...`
+
+Inspectez les requêtes de proxy, l’utilisation, le stockage, la mémoire et les données de débogage. Les alias directs sont :
+
+| Alias ​​| Ressource équivalente |
+| --- | --- |
+| `ocx logs [filters] [--follow] [--json|--jsonl]` | `ocx observe logs` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)